src/zodbupdate/convert.py

```diff
@@ -4,13 +4,16 @@
 import sys
 import types
 
-import pkg_resources
-
 import zodbpickle
 
 from zodbupdate import utils
 
 
+if sys.version_info.major == 3 and sys.version_info.minor < 10:
```

Comment:
```suggestion
if sys.version_info.major == 3 and sys.version_info.minor < 10:  # PY3.9
```

This makes it easier to get rid of this code once Python 3.9 support is dropped.